requirements/base.txt

```diff
@@ -6,7 +6,7 @@ Werkzeug<2.1
 #Need to add Schemas latest SDK.
 boto3>=1.19.5,==1.*
 jmespath~=0.10.0
-PyYAML~=5.3
+PyYAML>=5.4.1,==5.*
```

Comment:
Do we need to re-run `make update-reproducible-reqs`?

Reply:
I ran it there are no changes

Reply:
Also therefore no changes to the third party licenses